vsts/pipelines/_integration.yml

```diff
@@ -0,0 +1,160 @@
+parameters:
+  isCIBuild: 'false'
+
+jobs:
+- job: Job_RuntimeImages
+  displayName: Build and Test Runtime Images
+  condition: succeeded()
+  pool:
+    name: OryxLinux
+  # Building runtime images can take a long time due our PHP images
+  timeoutInMinutes: 100
+  steps:
+  - script: |
+      echo "##vso[task.setvariable variable=BuildBuildImages;]false"
+      echo "##vso[task.setvariable variable=TestBuildImages;]false"
+      echo "##vso[task.setvariable variable=BuildRuntimeImages;]true"
+      echo "##vso[task.setvariable variable=TestRuntimeImages;]true"
+      echo "##vso[task.setvariable variable=PushRuntimeImages;]true"
+      echo "##vso[task.setvariable variable=PushBuildImages;]false"
+      echo "##vso[task.setvariable variable=EmbedBuildContextInImages;]true"
+    displayName: 'Set variables'
+  - template: _buildParallel.yml
+    parameters:
+      pushToDockerHub: ${{ parameters.isCIBuild }}
+
+- job: Job_PythonIntegrationTests
```

Comment:
q: Why can't we move these tests tasks to the template so we can reuse them on every build? that is, make the tests parallel everywhere we run them.

Reply:
yes im working on this .. I created this PR on a stable point where I know it's working :)  … I will be cleaning up this now and update few more things